eth2/beacon/common/spec.go

```diff
@@ -108,6 +108,9 @@ type BellatrixPreset struct {
 	MAX_EXTRA_DATA_BYTES                       uint64 `yaml:"MAX_EXTRA_DATA_BYTES" json:"MAX_EXTRA_DATA_BYTES"`
 }
 
+type CapellaPreset struct {
```

Comment:
We should add these presets: https://github.com/ethereum/consensus-specs/blob/dev/presets/mainnet/capella.yaml

Reply:
Thanks! I was looking at the empty files of the master branch and missed these files.